docs/user/Build.md

```diff
@@ -0,0 +1,54 @@
+# Build OpenROAD Flow
+
+:::{warning}
+On CentOS 7 you need to manually make sure the PATH variable includes at
+least one of the new version of GCC/Clang. To enable GCC-8 or Clang-7 you
+need to run:
+
+```shell
+# to enable gcc-8
+source /opt/rh/devtoolset-8/enable
+# or to enable clang-7
+source /opt/rh/llvm-toolset-7.0/enable
+```
+:::
```

Comment:
<img width="391" alt="image" src="https://user-images.githubusercontent.com/9001905/214110837-89c0e7af-b9fc-4ec9-b86f-00b6141b1c5a.png">

The warning marker does not show well on GH.